api/routes/units.js

```diff
@@ -0,0 +1,159 @@
+const router = require('express').Router()
+const Unit = require('../models/units')
+const Company = require('../models/companies')
+// const Employee = require('../models/employees')
+
+// GET /api/units
+// Return a list of all of the units with all related information.
+router.get('/',(req, res, next) => {
+    const status = 200
+    Unit.find().then(response => {
+        console.log(response)
+        res.json({ status, response })
+    })
+})
+
+// GET /api/units?kind=[kind]
+// * e.g. `GET http://localhost:5000/api/units?kind=desk`
+// Return a list of all units where the kind is equal to the provided kind. If none are found, return an empty array.
+
+router.get('/', async (req, res, next) => {
+    const status = 200;
+    try {
+        const response = await Unit.find().select('kind')
```

Comment:
This will just pull out the `kind` field. It won't do any filtering based on the `req.query`.